### ISSUE TITLE
split term: alopecia universalis

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#2840]
**Term to split:**
MONDO:0008757 alopecia universalis

**Name for new terms:** Note - when a term should be split, there should be 2+ new terms. Please list the new labels below. 
alopecia universalis
alopecia universalis congenita

**List properties that should be moved to new term:** Note- ORDO tends to represent the generic class. Most properties (such as synonyms and database cross references) will move to the generic term. Synonynms that end in "1", "2", etc. should move to subclass.
AUC: 
OMIM:203655  
UMLS: C1859877

Notes:
The congenita form of alopecia congenita suggests it is present from birth, a distinct presentation from the other 2 MIM numbers on ORPHA:701. 
Also note that Orpha:701 links to 3 MIM numbers, but the other 2 are not currently set as children of this concept in MONDO: MONDO:0007082, MONDO:0012551
Suggest setting Alopecia unversalis as parent to 3 concepts: MONDO:0007082, MONDO:0012551 and the new term alopecia universalis congenita, reflecting the OrphaNet concept relationship with unique MIM numbers.

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**Comment by sabrinatoro** [comment]
Hello @kanems 
I'm Sabrina, a new curator who recently joined the Mondo team.
We will address your issue as soon as possible. Thank you for your patience.
If this is a high priority, please let me know and we can prioritize it. Thank you!

**Comment by nicolevasilevsky** [comment]
Hi @kanems 
Looks like the [OMIM term](https://omim.org/entry/203655) has ALOPECIA UNIVERSALIS CONGENITA as the primary label and Alopecia universalis as a synonym (alternative title).

The [Orphanet class](https://www.ebi.ac.uk/ols/ontologies/ordo/terms?iri=http%3A%2F%2Fwww.orpha.net%2FORDO%2FOrphanet_701) also has a mutation in the HR gene, so I think it is equivalent to the OMIM term, although Orphanet does not have an exact mapping to that term.

I changed the xrefs on alopecia areata 1 and 2 to be direct siblings of ORPHA:701 Alopecia universalis, I don't think they are subtypes of that though, as they do not have the HR mutation (from what I can tell).

I renamed MONDO_0008757 to 'alopecia universalis congenita' to match OMIM.
